### ISSUE TITLE
Update chartOptions series with more/less series only updates existing series

### ISSUE THREAD

**Issue by alb90** [highcharts/highcharts-react-native#59]
Hi - I have a chart which has a split by option and depending on the split by can return a varying number of series. If, for example my chart has 2 series, and the new data has 3 series, after updating the chartOptions in the state, the 3rd series is not shown, despite the existing 2 series updating. 

I believe in the React Highcharts wrapper there's a oneToOne prop which force updates all series, is this not including in this package?

```
export default class Highcharts extends React.PureComponent {
  constructor(props) {
    super(props);
    this.state = {
      chartOptions: {
        ...,
        series:  this.props.data
      }
    };
  }

  componentDidUpdate(prevProps) {
    const { data } = this.props;
    if (!isEqual(data, prevProps.data)) {
      this.setState(prevState => ({
        chartOptions: {
          ...prevState.chartOptions,
          series: data
        }
      }));
    }
  }

  render() {
    return (
      <HighchartsReactNative
        useCDN={true}
        useSSL={true}
        styles={styles.container}
        webviewStyles={styles.container}
        options={this.state.chartOptions}
      />
    );
  }
}
});

```

**Comment by astron97** [comment]
@sebastianbochan This is a real issue. I have tried with chartUpdate pattern only to have no effect.

**Comment by astron97** [comment]
Say for example I have basic bar chart where each category has three bars, If i try changing these from three to two bars the chart simply shows the initial three bars whilst able to update individual y values.
![Screenshot_2020-04-29-16-49-59-1145623991](https://user-images.githubusercontent.com/45058613/80590590-ce789780-8a39-11ea-83d9-d8b19fba3067.png)
the source of data is like this following screenshot:
Am adding and removing the objects from array.
![image](https://user-images.githubusercontent.com/45058613/80590739-0da6e880-8a3a-11ea-87c3-3a99d9588329.png)
the update pattern is like the below screenshot:
![image](https://user-images.githubusercontent.com/45058613/80591043-9a51a680-8a3a-11ea-9ba7-6234edeb7d4c.png)
the optionsSwitcher() is called in componentDidUpdate which logs the updated series array whereas the chart does not show the change in no. of bars.

**Comment by sebastianbochan** [comment]
Im debugging the problem and will let you know about results soon.

**Comment by sebastianbochan** [comment]
The issue was caused by missing "oneToOne" param in `chart.update()`.

Fixed the issue in 2.2.1 version.

Please test and reopen when new issues will come.
